SDK/include/NDK/Widgets/ButtonWidget.hpp

```diff
@@ -29,24 +30,69 @@ namespace Ndk
 
 			void ResizeToContent() override;
 
+			inline const Nz::Color& GetColor() const;
+			inline const Nz::Color& GetCornerColor() const;
```

Comment:
I'm not sure about the "corner color" name as it defines more than one corner (half of the button actually), maybe secondaryColor would be, despite being less precise, more accurate?